editor/src/uuiui/radix-components.tsx

```diff
@@ -41,6 +42,8 @@ export interface DropdownMenuItem {
   checked?: boolean
   onSelect: () => void
   danger?: boolean
+  separator?: boolean
```

Comment:
I'd split the separator item into a different subtype, so whenever a separator is added `onSelect` and `label` doesn't have to be filled out with dummy values (like it is earlier in the PR)

Reply:
[`27673f2` (#6161)](https://github.com/concrete-utopia/utopia/pull/6161/commits/27673f208a6342b464689c83b8e8dd7dc7fd1c7b)